dataserver.js

```diff
@@ -0,0 +1,29 @@
+'use strict'; // eslint-disable-line strict
+require('babel-core/register');
+
+const arsenal = require('arsenal');
+const config = require('./lib/Config.js').default;
+const logger = require('./lib/utilities/logger').logger;
+
+if (config.backends.data === 'file') {
+    /**
+     * Configure the file paths for data if using the file
+     * backend. If no path provided, uses data at the root of
+     * the S3 project directory.
+     */
+    const dataPath = process.env.S3DATAPATH ?
```

Comment:
@rahulreddy I think Lauren was only talking about putting the ENV var reading all in one place, not to create a new configuration value in config.json, nevertheless I second you on this one, I think it's a good idea. We can still allow the possibility of an ENV var to override the config value, which would not break the existing setups.

We don't have the feature of "pulling config from one place" when it comes to reading the exact same file from different servers, but we can duplicate the config.json on both S3 connectors and the dmd so that the config is consistent (actually the dmd would not run if not provided a valid S3 config file since it's calling the global config parsing code). For the single-server, simple deployment it makes perfect sense to have only one config.json containing all the config.